priv/repo/migrations/20200521031600_create_tasks.exs

```diff
@@ -30,7 +30,7 @@ defmodule BikeBrigade.Repo.Migrations.CreateTasks do
       add :pickup_province_string, :string
       add :pickup_postal, :string
       add :pickup_country, :string
-      add :pickup_loacation, :geography
+      add :pickup_location, :geography
```

Comment:
This typo is ameliorated in priv/repo/migrations/20200521035759_rename_driver_notes_to_rider_notes.exs